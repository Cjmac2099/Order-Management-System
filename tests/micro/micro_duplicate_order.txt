WAGZ2AHLHK
35.66
WFSNHFN3BQ
21
BD7X2ZYQQL
59.01
WAGZ2AHLHK
13.72
END
