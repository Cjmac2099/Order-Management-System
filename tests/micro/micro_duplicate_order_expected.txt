Loading catalog...
Enter 'END' to finish order.
SKU> 
SKU matches product Juice Apples frozen concentrate, price per pound: $0.61
Quantity>
SKU> 
SKU matches product Juice Apples ready-to-drink, price per pound: $0.87
Quantity>
SKU> 
SKU matches product Canned Apples applesauce, price per pound: $1.17
Quantity>
SKU> 
SKU matches product Juice Apples frozen concentrate, price per pound: $0.61
Quantity>
SKU> 
Printing order summary:
35.66lbs Juice Apples frozen concentrate: $21.70
21.00lbs Juice Apples ready-to-drink: $18.27
59.01lbs Canned Apples applesauce: $69.07
13.72lbs Juice Apples frozen concentrate: $8.35
